pt-BR/getting_started.md

```diff
@@ -811,19 +811,19 @@ Visit <http://localhost:3000/articles/new> and give it a try!
 
 ![Show action for articles](images/getting_started/show_action_for_articles.png)
 
-### Listing all articles
+### Listando todos os artigos
 
-We still need a way to list all our articles, so let's do that.
-The route for this as per output of `rails routes` is:
+Nós ainda precisaremos de um jeito para listar todos nossos artigos, então vamos fazer isso..
+A rota para isso conforme a saída de `rails routes` é 
 
 ```
 articles GET    /articles(.:format)          articles#index
 ```
 
-Add the corresponding `index` action for that route inside the
-`ArticlesController` in the `app/controllers/articles_controller.rb` file.
-When we write an `index` action, the usual practice is to place it as the
-first method in the controller. Let's do it:
+Adicione a correspondente action  `index` para essa rota dentro de
```

Comment:
```suggestion
Adicione a *action* `index` correspondente a essa rota dentro do
```
Termos em inglês não traduzidos, como *action* e *controller*, devem vir em itálico. Nosso manual de tradução tem descritos os padrões que estamos utilizando no guia. :)